ledger-core/v2/runner/context.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/assured-ledger/blob/master/LICENSE.md.
+
+package runner
+
+import (
+	"github.com/pkg/errors"
+
+	"github.com/insolar/assured-ledger/ledger-core/v2/runner/execution"
+	descriptor "github.com/insolar/assured-ledger/ledger-core/v2/runner/executionevent"
+	"github.com/insolar/assured-ledger/ledger-core/v2/runner/executionupdate"
+	"github.com/insolar/assured-ledger/ledger-core/v2/runner/requestresult"
+)
+
+type executionContext struct {
```

Comment:
Execution is public part of system, it would be used by Runner to give information about what we need to execute contract.
executionContext is internal structure to have ability  to return back to execution (from outside). Probably bad naming, do you have better name?